sdk/python/generative-ai/rag/notebooks/db_copilot_with_rag.ipynb

```diff
@@ -0,0 +1,396 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "llm_completion_config = f'{{\"type\":\"azure_open_ai\",\"model_name\":\"{aoai_completion_model_name}\",\"deployment_name\":\"{aoai_completion_deployment_name}\",\"temperature\":0,\"max_tokens\":\"1500\"}}'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "56878876",
+   "metadata": {},
+   "source": [
+    "### Setup Pipeline\n",
+    "\n",
+    "The Components are published to a [Registry](https://learn.microsoft.com/azure/machine-learning/how-to-manage-registries?view=azureml-api-2&tabs=cli), `azureml`, which should have access to by default, it can be accessed from any Workspace.\n",
+    "In the below cell we get the Component Definitions from the `azureml` registry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e2a3752a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_registry = MLClient(credential=credential, registry_name=\"azureml\")\n",
+    "\n",
+    "db_copilot_component = ml_registry.components.get(\n",
+    "    \"llm_ingest_db_to_faiss\", label=\"latest\"\n",
+    ")\n",
+    "\n",
+    "print(db_copilot_component)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "209700d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.dsl import pipeline\n",
+    "\n",
+    "\n",
+    "@pipeline(name=f\"db_copilot_vector_pipeline_faiss\", default_compute=\"serverless\")\n",
+    "def db_copilot_vector_pipeline_faiss(\n",
+    "    aoai_connection: str,\n",
+    "    db_datastore: str,\n",
+    "    embeddings_model: str,\n",
+    "    chat_aoai_deployment_name: str,\n",
+    "    embedding_aoai_deployment_name: str,\n",
+    "    mlindex_dataset_name: str,\n",
+    "    selected_tables: str = None,\n",
+    "    max_sampling_rows: int = 3,\n",
+    "):\n",
+    "    db_copilot_component(\n",
+    "        db_datastore=db_datastore,\n",
+    "        embeddings_model=embeddings_model,\n",
+    "        chat_aoai_deployment_name=chat_aoai_deployment_name,\n",
+    "        embedding_aoai_deployment_name=embedding_aoai_deployment_name,\n",
+    "        embeddings_dataset_name=mlindex_dataset_name,\n",
+    "        embedding_connection=aoai_connection,\n",
+    "        llm_connection=aoai_connection,\n",
+    "        selected_tables=selected_tables,\n",
+    "        max_sampling_rows=max_sampling_rows,\n",
+    "    )\n",
+    "    return {}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6682ea80",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datastore_name = \"<test_db_datastore_name>\"\n",
+    "aoai_embedding_model_name = \"text-embedding-ada-002\"\n",
+    "asset_name = \"llm_index_db_dataset\"\n",
+    "pipeline_job = db_copilot_vector_pipeline_faiss(\n",
+    "    aoai_connection=aoai_connection_id,\n",
+    "    db_datastore=f\"azureml://datastores/{datastore_name}\",\n",
```

Comment:
Ideally link clearly to a doc that shows how to setup Database Datastore properly

Reply:
I can not find a public doc to create sql db in UI.

Reply:
Then a quick python/markdown cell that explains fields to set? Or uses V1 SDK to do it? Otherwise not sure how users will connect their data.